modules/distributor/receiver/shim.go

```diff
@@ -248,22 +250,16 @@ func newLogger(level logging.Level) *zap.Logger {
 	return logger
 }
 
-func newMetricViews() ([]*view.View, error) {
-	views := obsreport.Configure(configtelemetry.LevelNormal)
-	err := view.Register(views...)
```

Comment:
We need to register views in order to get metrics from the OTel components. Otherwise, the registered exporter below won't have views to export.

Unfortunately, the `obsmetrics` pkg has been made internal, so we can't import it here. We should find a replacement or solution to it.

Reply:
Hi @mapno, thanks for the feedback

I am not sure if keeping the old metrics from the opentelemetry collector is still an option, they mention they want to expose opentelemetry metrics instead of opencesus but I believe that is not an option yet. 

Do you suggest to scrap the registered exporter or implementing the metrics ourrselves?

Reply:
@mapno should we use Grafana's otel collector fork for this?

Reply:
Right, I don't think there is a direct solution at the moment that doesn't involve a fork, implementing the metrics ourselves or a similar approach.

At Grafana we currently host a [fork](https://github.com/grafana/opentelemetry-collector/) of the collector in which `/internal` folders are exported, so we can circumvent this issue.

You can see the Agent importing this fork: [go.mod](https://github.com/grafana/agent/blob/main/go.mod#L140-L143). To get PR out, I'm fine with using this updating and using the fork for v0.40.0, while a solution is found. Does that sound good to you @tete17
?

Reply:
This [branch](https://github.com/grafana/opentelemetry-collector/tree/0.40-grafana) is updated to v0.40.0 with internal folders exported

Reply:
Sounds like an awesome plan.

Reply:
This feels like a much deeper conversation that what I intended to go with a quick and dirty dependency update :laughing: